pkg/api/pod/util.go

```diff
@@ -1373,3 +1379,20 @@ func useOnlyRecursiveSELinuxChangePolicy(oldPodSpec *api.PodSpec) bool {
 	// No feature gate + no value in the old object -> only Recursive is allowed
 	return true
 }
+
+// restartableInitContainersResizePolicy returns true if the pod spec is non-nil and
+// it has any init container with ContainerRestartPolicyAlways and non-nil ResizePolicy.
+func restartableInitContainersResizePolicy(podSpec *api.PodSpec) bool {
```

Comment:
nit: rename for consistency
```suggestion
func hasRestartableInitContainerResizePolicy(podSpec *api.PodSpec) bool {
```